compiler/rustc_llvm/llvm-wrapper/RustWrapper.cpp

```diff
@@ -116,6 +116,35 @@ extern "C" LLVMValueRef LLVMRustGetNamedValue(LLVMModuleRef M, const char *Name,
   return wrap(unwrap(M)->getNamedValue(StringRef(Name, NameLen)));
 }
 
+enum class LLVMRustTailCallKind {
+  None,
+  Tail,
+  MustTail,
+  NoTail,
+  Last,
```

Comment:
[TCK_LAST](https://llvm.org/doxygen/classllvm_1_1CallInst.html#ad682514c13f12f1a8d759d422fce6aefacbe2d4161d70f460739c3aa3a19662aa) is numerically identical to the highest enum member (TCK_NoTail). It's used for loop bounds, not an actual meaningful value.

Reply:
Ooops 😅